Documentation/CRDs/Block-Storage/ceph-block-pool-crd.md

```diff
@@ -182,57 +182,56 @@ spec:
 
 ### Spec
 
-* `replicated`: Settings for a replicated pool. If specified, `erasureCoded` settings must not be specified.
+*   `replicated`: Settings for a replicated pool. If specified, `erasureCoded` settings must not be specified.
     * `size`: The desired number of copies to make of the data in the pool.
     * `requireSafeReplicaSize`: set to false if you want to create a pool with size 1, setting pool size 1 could lead to data loss without recovery. Make sure you are *ABSOLUTELY CERTAIN* that is what you want.
-    * `replicasPerFailureDomain`: Sets up the number of replicas to place in a given failure domain. For instance, if the failure domain is a datacenter (cluster is
-stretched) then you will have 2 replicas per datacenter where each replica ends up on a different host. This gives you a total of 4 replicas and for this, the `size` must be set to 4. The default is 1.
-    * `subFailureDomain`: Name of the CRUSH bucket representing a sub-failure domain. In a stretched configuration this option represent the "last" bucket where replicas will end up being written. Imagine the cluster is stretched across two datacenters, you can then have 2 copies per datacenter and each copy on a different CRUSH bucket. The default is "host".
-* `erasureCoded`: Settings for an erasure-coded pool. If specified, `replicated` settings must not be specified. See below for more details on [erasure coding](#erasure-coding).
+    * `replicasPerFailureDomain`: Sets up the number of replicas to place in a given failure domain. For instance, if the failure domain is a datacenter (cluster is stretched) then you will have 2 replicas per datacenter where each replica ends up on a different host. This gives you a total of 4 replicas and for this, the `size` must be set to 4. The default is 1.
+    * `subFailureDomain`: Name of the CRUSH bucket representing a sub-failure domain. In a stretched configuration this option represent the "last" bucket where replicas will end up being written. Imagine the cluster is stretched across two datacenters, you can then have 2 copies per datacenter and each copy on a  different CRUSH bucket. The default is "host".
+*   `erasureCoded`: Settings for an erasure-coded pool. If specified, `replicated` settings must not be specified. See below for more details on [erasure coding](#erasure-coding).
```

Comment:
Are the spaces between the `*` and first word intentional? That doesn't seem expected to me.

Reply:
Yes. We can chat more about some of the weird things that this is going to be requiring like this one.

Reply:
markdownlint really isn't all that great as a tool. It particularly has gaps in requiring 4-space tabs throughout the full document. It will only force it on list items but not the paragraph conentents of list items.

Rook uses **a lot** of lists with multiple paragraphs and code blocks inside lists, which mkdocs won't render properly unless those are indented very exactly. 

I had to find something else that we could use to help fill in this particular gap. The tool called "Prettier" was commonly recommended for markdown, and I found that using its `--tab-width=4` option helps to ensure that paragraph contents (especially code blocks) are spaced appropriately to be rendered by mkdocs and/or flagged as an error by markdownlint. 

The downside is that Prettier is VERY opinionated. It requires using `-` for bullet points, and it requires that bullet points adhere to the tab-width option. It also doesn't like the standard callout/admonition syntax and requires a newline between the callout header and text.

IMO, this is certainly not ideal, but it is much more usable than having docs that render properly on github but silently fail (and create subtle bugs) in user-facing docs.